src/apis.js

```diff
@@ -13,6 +13,13 @@ const chainsferApi = axios.create({
   baseURL: env.REACT_APP_CHAINSFER_API_ENDPOINT,
   headers: {
     'Content-Type': 'application/json'
+  },
+  validateStatus: status => {
```

Comment:
The server maintenance status checking should be done here: 
https://github.com/ventureum/Chainsfr/blob/master/src/App.js#L190

Reply:
preload can only check when refresh